kuma/javascript/src/ssr.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+
+import CurrentUser from './current-user.jsx';
+import DocumentProvider from './document-provider.jsx';
+import Page from './page.jsx';
+
+/*
+ * This function performs server-side rendering of our UI, given
+ * a JSON object of document data. It is used by ../ssr-server.js
+ */
+export default function ssr(data) {
+    return renderToString(
+        <DocumentProvider initialDocumentData={data}>
+            <CurrentUser.Provider>
+                <Page />
+            </CurrentUser.Provider>
+        </DocumentProvider>
+    );
```

Comment:
Is there any chance you can merge these file and index.jsx to avoid the DRY?
Today they're both pretty simple but might outgrow itself and someone might refactor `Page` to become `DocumentPage` and get confused that they had to edit one other file too.

Reply:
Yes, I've been meaning to create a top-level App component that just renders this DocumentProvider/CurrentUser/Page thing. But I didn't want to complicate this PR by doing it now.

Reply:
Fine by me. Perhaps put one of those `// TODO: refactor this bla bla bla` :)